sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/aspnethostedservice/Directory.Build.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <!-- Use decentralized package references when building outside https://github.com/Azure/azure-sdk-for-net -->
+  <ItemGroup Condition="'$(IsSamplesProject)' != 'true'">
+    <PackageReference Update="Azure.Messaging.EventHubs.Processor" Version="5.10.0" />
+  </ItemGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.targets))\Directory.Build.targets"
+          Condition="'$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.targets))' != ''" />
```

Comment:
We don't want the condition here; this will need to link back up to the root targets file in order to keep the needed integration with the repo engineering process and systems.   We'll need to adjust the path to climb up a few more subdirectories.